www/src/input/tutorial.md

```diff
@@ -0,0 +1,1790 @@
+## [Installation](#installation) {#installation}
```

Comment:
Is the "Welcome to Roc! This tutorial will teach you how to build Roc applications [...]." section still included?

Reply:
Yes, but it is in the `tutorial.roc` instead of markdown. It's there so the formatting still looks acceptable with the current css. I'll have another look and see if I can move it all into markdown.